TEST-PLAN.md

```diff
@@ -0,0 +1,109 @@
+# DP Creator II Test Plan
+
+## Introduction
+
+### Purpose
+
+Automate the end-to-end testing of the application to ensure it meets user requirements and maintains reliability, especially in the context of privacy-preserving data processing.
+
+### Project Overview
+
+The single user application will be installed with pip, and offer a web application (running on localhost) that guides the user through the application of differential privacy on a data file they provide.
```

Comment:
"Single user application" is a phrase I haven't heard much. I think I know what you mean but perhaps this could be simplified to just "application".